condacolab.py

```diff
@@ -39,20 +41,46 @@
 __author__ = "Jaime Rodríguez-Guerra <jaimergp@users.noreply.github.com>"
 
 
-PREFIX = "/usr/local"
+PREFIX = "/opt/miniconda"
```

Comment:
```suggestion
PREFIX = "/opt/conda"
```

Let's make this just `/opt/conda` to not introduce a bias on what distribution was installed 🤞